x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,344 @@
+                    <EuiDescriptionListTitle>
+                      <FormattedMessage
+                        id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.eligibleTitle"
+                        defaultMessage="These jobs are available if data exists that match the following query:"
+                      />
+                    </EuiDescriptionListTitle>
+                    <EuiSpacer size="m" />
+                    <EuiDescriptionListDescription>
+                      <EuiCodeBlock
+                        language="json"
+                        isCopyable
+                        overflowHeight="500px"
+                        data-test-subj="mlPreconfigJobsQueryBlock"
+                      >
+                        {JSON.stringify(module.query, null, 2)}
+                      </EuiCodeBlock>
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check data views to determine which can be used to create jobs with this configuration."
+                          />
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        {canCreateJob === false ? (
+                          <EuiToolTip
+                            position="top"
+                            content={createPermissionFailureMessage('canCreateJob')}
+                          >
+                            {runRecognizerButton}
+                          </EuiToolTip>
+                        ) : (
+                          runRecognizerButton
+                        )}
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                ) : null}
+                {recognizerWasRun ? (
+                  <EuiFlexItem grow={false}>
+                    <DataViewsTable
+                      matchingDataViews={matchingDataViews}
+                      moduleId={module.id}
+                      jobsLength={module.jobs.length}
+                    />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            ) : (
+              <EuiEmptyPrompt
+                css={{ margin: `${euiTheme.size.s}` }}
+                color="primary"
+                iconType={isLogoObject(module.logo) ? module.logo?.icon : undefined}
```

Comment:
nit,
```suggestion
                iconType={isLogoObject(module.logo) ? module.logo.icon : undefined}
```

Reply:
👍  Updated in 7cba584108770e2ce155348b8af92713b74b3f43